felles/log/src/main/java/no/nav/vedtak/log/audit/Auditdata.java

```diff
@@ -0,0 +1,33 @@
+package no.nav.vedtak.log.audit;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Data som utgjør et innslag i sporingsloggen i "Common Event Format (CEF)".
+ */
+public class Auditdata {
+    
+    private static final String FIELD_SEPARATOR = " ";
+    
+    private AuditdataHeader header;
+    private Set<CefField> fields;
+
+    public Auditdata(AuditdataHeader header, Set<CefField> fields) {
+        this.header = header;
+        this.fields = new HashSet<>(fields);
```

Comment:
kunne brukt Set.copyOf for å gjøre den immutable?  Vil det være en fordel her om feltene er stabilt sortert når outputtes til CEF format? (eks. for senere søk, regex, eller annet?  I såfall bør bruke TreeSet?

Reply:
Jeg sorterer før toString. Dermed er rekkefølgen alltid deterministisk. "fields" er ikke eksponert -- dermed er den immutable.

Reply:
sløvt av meg 👍